src/point_one/fusion_engine/messages/defs.h

```diff
@@ -80,6 +80,8 @@ enum class MessageType : uint16_t {
   GET_MESSAGE_RATE = 13221, ///< @ref GetMessageRate
   MESSAGE_RATE_RESPONSE = 13222, ///< @ref MessageRateResponse
 
+  SYSTEM_STATUS = 13300, ///< @ref SystemStatusMessage
```

Comment:
I'd suggest we make group this under 10500+ (below solution messages above) to give it more breathing room, and add a comment to separate device messages like:
```
// Device status messages.
SYSTEM_STATUS = 10500, ///< @ref SystemStatusMessage
```

And matching changes to `defs.py`